ibis/common/collections.py

```diff
@@ -150,6 +150,16 @@ def __eq__(self, other):
         return dict(self.items()) == dict(other.items())
 
 
+class ConflictingValuesError(ValueError):
```

Comment:
```suggestion
class ConflictingValuesError(Exception):
```

I understand this breaks code that looks like `except ValueError:` but our exception types are not (yet) part of the public API.

Reply:
Actually nevermind, let's leave it as a subclass for now.